app/views/notes/index.html.erb

```diff
@@ -12,12 +12,12 @@
   </thead>
 
   <tbody>
-    <% @notes.each do |note| %>
+    <% !@notes.empty? && @notes.each do |note| %>
```

Comment:
Just curious, but why the check for `empty?` here? Surely if it's empty it will simply not iterate?

Reply:
If it's nil, then it will show a missing method